### PR TITLE
Buy Shirt

### PR DESCRIPTION
**Pull request by SebastianGomezVidal**
It was working yesterday, it seems there are lot of traffic in the page today.

### CODE REVIEW

**Comment by SebastianGomezVidal** [comment]
Screen Shot 2022-07-21 at 2 55 27 PM

**Comment by SebastianGomezVidal** [comment]
Screen Shot 2022-07-21 at 3 10 08 PM

File: cypress/e2e/buy-shirt.cy.ts [hunk 1]
```diff
@@ -0,0 +1,40 @@
+describe("Buy a t-shirt", () => {
+  it("then the t-shirt should be bought", () => {
+    // Go to homepage
```
Comment (danielgalvis98) [inline cypress/e2e/buy-shirt.cy.ts:3]: Please try to avoid to put all this comments, in every line, it affects readability

Comment (SebastianGomezVidal) [reply cypress/e2e/buy-shirt.cy.ts:3]: Hello Daniel, is the way to track the actions to be performed.

Comment (manuelq12) [reply cypress/e2e/buy-shirt.cy.ts:3]: @SebastianGomezVidal Its understandable that you want to keep the comments, but its only due to the fact that the code is not self-explanatory. Next time, there will be no approval will be done until the comments are removed as this is not a good development practice.

**Review by manuelq12 — APPROVED** [review]

**Pull request merged**